docs: add upgrade page & update getting started (#3439)
coder/coder: docs/configure.md

# Configure

> This article documents Coder's primary configuration variables. For a full list

Reviewer: ```suggestion
This article documents Coder server's primary configuration variables. For a full list
```

of the Coder environment variables, run `coder server --help` on the host.

> ## Coder server configuration

Reviewer: ```suggestion
```
This is redundant.

> Once you've installed Coder, you can configure the server by setting the following

Reviewer: ```suggestion
Once you've [installed Coder](../install.md), you can configure the server by setting the following
```
